en/asgardeo/docs/references/service-extensions/in-flow-extensions/custom-authentication/api/custom-authentication-action-v1.yaml

```diff
@@ -83,6 +83,8 @@ paths:
                           value: johnd@gmail.com
                         - uri: http://wso2.org/claims/emailaddress
                           value: johnd@gmail.com
+                        - uri: http://wso2.org/claims/multiValuedClaim
```

Comment:
Here shall we give an exact wso2 specific multi valued claim. Because we say claim uri should be in wso2 dialect